dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPIImpl.java

```diff
@@ -466,23 +466,84 @@ public void deleteVersionByInode(final String inode) {
 		Try.run(()->FactoryLocator.getTemplateFactory().deleteTemplateByInode(inode)).onFailure(e -> new RuntimeException(e));
 	}
 
+	/*
```

Comment:
I would add integration tests to cover this logic. There are some validations here that aren't considered in the postman test included

Reply:
done